config/defaults.mk

```diff
@@ -46,6 +46,10 @@ CUDA_FLAGS = -x=cu --expt-extended-lambda -arch=$(CUDA_ARCH)
 CUDA_XCOMPILER = -Xcompiler=
 CUDA_XLINKER   = -Xlinker=
 
+# HIP configuration options
+HIP_CXX = hipcc
+HIP_FLAGS = --amdgpu-target=gfx900 # link-time option
```

Comment:
So that needs to change for a different GPU model?

Reply:
Yes, and I added these comments:
```
# The 'amdgpu' option allows to specify the AMD GPU processor:
# gfx600:tahiti, gfx700:kaveri, gfx701:hawaii, gfx801:carrizo, gfx900, gfx1010
```